frontend/app/assets/javascripts/spree/frontend/checkout/address.js

```diff
@@ -43,6 +43,7 @@ Spree.ready(function($) {
       var statePara = $("#" + region + "state");
       var stateSelect = statePara.find("select");
       var stateInput = statePara.find("input");
+      var stateSpanRequired = statePara.find('[id$="state-required"]');
```

Comment:
Where is this var used?

Reply:
mhhh we've probably had some change in that file in the meantime, or I missed something while rebasing. I honestly can't recall why that var was required, I'll investigate soon, thanks :)